lib/oban.ex

```diff
@@ -505,4 +541,18 @@ defmodule Oban do
   end
 
   defp child_name(name, child), do: Module.concat(name, child)
+
+  defp send_signal(name, payload) do
+    name
+    |> child_name("Midwife")
+    |> send({:notification, :signal, json_encode_decode(payload)})
```

Comment:
This will technically work, but I think adding an api function directly to the `Midwife` module would be clearer. Something like `start_queue/2` and `stop_queue/2`, which then can handle sending a message within the module.

Reply:
That’s a great suggestion and I agree that it will more clear after putting the Client API functions in `Oban.Midwife`.